app/modules/notifications/models.py

```diff
@@ -67,6 +72,24 @@ class NotificationChannel(str, enum.Enum):
         'email_digest_content_template': 'collaboration_edit_request_digest.jinja2',
         'mandatory_fields': {'collaboration_guid'},
     },
+    NotificationType.merge_request: {
```

Comment:
I'm thinking this should be an individual_merge_request to:
1 be unambiguous 
2 Not require rework if we ever decide we want to merge something else

Reply:
agreed; i have done this as well as method change you mention below.